driver/bpf/fillers.h

```diff
@@ -5795,8 +5795,8 @@ FILLER(sys_bpf_x, true)
 	bpf_push_s64_to_ring(data, fd);
 
 	/* Parameter 2: cmd (type: PT_INT32) */
-	int32_t cmd = (int32_t)bpf_syscall_get_argument(data, 0);
-	return bpf_push_s32_to_ring(data, cmd);
+	unsigned long cmd = bpf_syscall_get_argument(data, 0);
```

Comment:
`cmd` is `int` why are we changing this? https://man7.org/linux/man-pages/man2/bpf.2.html